src/common/components/CoolerCard/CoolerCard.tsx

```diff
@@ -22,37 +22,38 @@ interface IProps {
 export default class CoolerCard extends Component<IProps> {
 	public render() {
 		const {cooler} = this.props;
+		const {title, series, roomID, place, source, type, certification, cop, power} = cooler;
 
 		return (
 			<ScrollView>
 				<View style={styles.titleWrapper}>
-					<Text style={styles.title}>{cooler.title}</Text>
-					<Text style={styles.series}>{cooler.series}</Text>
-					<Text style={styles.room}>Room: {cooler.room}</Text>
+					<Text style={styles.title}>{title}</Text>
+					<Text style={styles.series}>{series}</Text>
+					<Text style={styles.room}>Room: {(roomID !== null) ? roomID : 'No Room'}</Text>
 				</View>
 				<View style={styles.optionsWrapper}>
```

Comment:
looks like the same:
```
const items = [
   {
       title: 'Place of Origin',
       type: source,
   },
   ...
]

items.map(item => (
     <View style={styles.optionsWrapper}>
	   <Text style={styles.option}>{title}</Text>
	   <Text style={styles.description}>{type}</Text>
     </View>
))```

Reply:
because it's looks cleaner than your way, and it's expandable

Reply:
done